.github/workflows/pr-automation.yml

```
@@ -0,0 +1,186 @@
+          <!-- 관련 이슈가 있다면 링크해주세요 -->
+          Closes #"
+          
+          # 멀티라인 출력
+          {
+            echo 'body<<EOF'
+            echo "$PR_BODY"
+            echo 'EOF'
+          } >> $GITHUB_OUTPUT
+
+      - name: Update PR Body
+        uses: peter-evans/create-or-update-comment@v3
```

actionlint (1.7.11)

[error] the runner of "peter-evans/create-or-update-comment@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)